Dollar_convert.sas

```diff
@@ -160,7 +161,8 @@
 	%let CPI_2016 = 240.007;  %** Annual 2016 **;
 	%let CPI_2017 = 245.120;  %** Annual 2017 **;
 	%let CPI_2018 = 251.107;  %** Annual 2018 **; 
-	%let CPI_2019 = 254.412;  %** first half average **;  
+	%let CPI_2019 = 255.658;  %** Annual 2019 **;  
+	%let CPI_2020 = 258.255;  %** 2020-Q1 **; 
```

Comment:
Not sure how you came up with the quarterly number. Did you average together Jan, Feb, Mar? I cannot match your value.